webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJob.java

```diff
@@ -0,0 +1,153 @@
+package com.box.l10n.mojito.entity;
+
+import com.box.l10n.mojito.json.ObjectMapper;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobProperties;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import java.util.Date;
+import org.hibernate.envers.Audited;
+
+@Audited
+@Entity
+@Table(name = "scheduled_job")
+public class ScheduledJob {
+  @Id private String id;
+
+  @ManyToOne
+  @JoinColumn(name = "repository_id")
+  private Repository repository;
+
+  @ManyToOne
+  @JoinColumn(name = "job_type")
+  private ScheduledJobTypeEntity jobType;
+
+  @Column(name = "cron")
+  private String cron;
+
+  @Transient private ScheduledJobProperties properties;
+
+  @Column(name = "properties")
+  private String propertiesString;
+
+  @ManyToOne
+  @JoinColumn(name = "job_status")
+  private ScheduledJobStatusEntity jobStatus;
+
+  @Column(name = "start_date")
+  private Date startDate;
```

Comment:
We want to store date information in the DB in a timezone-aware away, so we generally use ZonedDateTime to achieve that.

Reply:
Fixed @garionpin